### DISCUSSION TITLE
After updating to 0.18.2, no heatmap and some functions missing/

### DISCUSSION THREAD

**Discussion by dwong33** [usememos/memos#2711] [General]
I appreciate the most updated version 0.18.2, it reorganizes the structure of memos, the personal page is getting much better.

But I observe that the heatmap was replaced by the "Statistic" panel, I think the heatmap function can be retained, or make it an optional function that can be enabled in settings. For it is a more visualized way of showing the number of memos.

I also appreciate the previous versions that there's the id of memos, it is now gone. For this function is important when we are 'marking' other memos. Can we reintroduce that?

And currently, for a non-admin user, there's no longer an indication showing that the platform is powered by memos. Previously, it was under the dropdown of the avatar, but now that dropdown is simplified. I will suggest including the 'powered by memos' in the settings page. 

Also, when will the weblate translation reopens? There are some translations yet to be included.

Thank you.

**Comment by chengguangnan** [comment]
确实，我也在寻找一个方法，去隐藏 Statistic panel，因为我不关心这些数据。

还有就是，左侧的菜单栏，现在整体的一个灰色，也比过去更*沉重*了。之前是中间的内容浮动在灰色背景上的感觉，让人更加关注内容。现在左侧的菜单栏浮动在最上层的感觉。

之前右边的 Heatmap 绿色，整个页面也稍微显得有点活力。现在灰色文字的 Statistic，有点信息过载的感觉。

比如 Profile 这个按钮，过去是在 logo 的弹出菜单里，其实我很少会去修改自己的 Profile，所以其实放在这里也有点碍事的感觉。

再比如 Timeline 这个按钮，对我而言似乎和 Home 界面一样，没有什么功能性。

还有两个版本都有的 Explore 对于单机单用户应该也是没有什么用的。

我还是切回 0.18.1 了。
